lib/tern.js

```diff
@@ -660,9 +660,14 @@
     var typeInfo = query.types || query.docs || query.urls || query.origins;
     var wrapAsObjs = typeInfo || query.depths;
 
-    for (var i = 0; i < completions.length; ++i) {
+    for (var i = 0, len = completions.length; i < len; i++) {
```

Comment:
This is an [unnecessary](https://www.eternalcoding.com/?p=33) optimization which has nothing to do with the actual change you're making in this patch.